app/templates/main/help-links.html

```diff
@@ -0,0 +1,25 @@
+{% macro helpLinks() %}
+
+    <div>
+      <h3 class="govuk-heading-s">Service Desk</h3>
+        <p class="govuk-body"><a href="#" class="govuk-link">Submit a request for help</a></p>
+        <p class="govuk-body">We aim to respond within 1 to 2 working days</p>
```

Comment:
Looks like there is currently too much spacing between these two in comparison with the Figma -possibly these two lines should be in the same paragraph but separated with a new line.